boards/limifrog-v1/include/periph_conf.h

```diff
@@ -0,0 +1,243 @@
+#define UART_0_PORT         GPIOC
+#define UART_0_PORT_CLKEN() (RCC->AHBENR |= RCC_AHBENR_GPIOCEN)
+#define UART_0_RX_PIN       11
+#define UART_0_TX_PIN       10
+#define UART_0_AF           7
+
+/* UART 1 device configuration */
+#define UART_1_DEV          USART3        /* BTLE */
+#define UART_1_CLKEN()      (RCC->APB2ENR |= RCC_APB2ENR_USART1EN)
+#define UART_1_CLK          (CLOCK_CORECLOCK)
+#define UART_1_IRQ          USART1_IRQn
+#define UART_1_ISR          isr_usart1
+/* UART 1 pin configuration */
+#define UART_1_PORT         GPIOA
+#define UART_1_PORT_CLKEN() (RCC->AHBENR |= RCC_AHBENR_GPIOAEN)
+#define UART_1_RX_PIN       10
+#define UART_1_TX_PIN       9
+#define UART_1_AF           7
+/** @} */
+
+/**
+ * @brief SPI configuration
+ * @{
+ */
+#define SPI_NUMOF           (2U)
+#define SPI_0_EN            1
+#define SPI_1_EN            1
+
+/* SPI 0 device configuration */
+#define SPI_0_DEV           SPI1  /* Used for OLED Display and external pins */
+#define SPI_0_CLKEN()       (RCC->APB2ENR |= RCC_APB2ENR_SPI1EN)
+#define SPI_0_CLKDIS()      (RCC->APB2ENR &= ~(RCC_APB2ENR_SPI1EN))
+#define SPI_0_IRQ           SPI1_IRQn
+#define SPI_0_ISR           isr_spi1
+/* SPI 0 pin configuration */
+#define SPI_0_PORT_CLKEN()  (RCC->AHBENR |= RCC_AHBENR_GPIOAEN)
+#define SPI_0_PORT          GPIOA
+#define SPI_0_PIN_SCK       5
+#define SPI_0_PIN_MOSI      7
+#define SPI_0_PIN_MISO      6
+#define SPI_0_PIN_AF        5
+
+/* SPI 1 device configuration */
+#define SPI_1_DEV           SPI3          /* Adesto AT45DB641E */
+#define SPI_1_CLKEN()       (RCC->APB1ENR |= RCC_APB1ENR_SPI3EN)
+#define SPI_1_CLKDIS()      (RCC->APB1ENR &= ~(RCC_APB1ENR_SPI3EN))
+#define SPI_1_IRQ           SPI3_IRQn
+#define SPI_1_ISR           isr_spi3
+/* SPI 1 pin configuration */
+#define SPI_1_PORT_CLKEN()  (RCC->AHBENR |= RCC_AHBENR_GPIOBEN)
+#define SPI_1_PORT          GPIOB
+#define SPI_1_PIN_SCK       3
+#define SPI_1_PIN_MOSI      5
+#define SPI_1_PIN_MISO      4
+#define SPI_1_PIN_AF        6
+/** @} */
+
+/**
+ * @name I2C configuration
+  * @{
+ */
+#define I2C_NUMOF           (2U)
+#define I2C_0_EN            1
+#define I2C_1_EN            1
+#define I2C_IRQ_PRIO        1
+#define I2C_APBCLK          (36000000U)
+
+/* I2C 0 device configuration */
+#define I2C_0_DEV           I2C1
+#define I2C_0_CLKEN()       (RCC->APB1ENR |= RCC_APB1ENR_I2C1EN)
+#define I2C_0_CLKDIS()      (RCC->APB1ENR &= ~(RCC_APB1ENR_I2C1EN))
+#define I2C_0_EVT_IRQ       I2C1_EV_IRQn
+#define I2C_0_EVT_ISR       isr_i2c1_ev
+#define I2C_0_ERR_IRQ       I2C1_ER_IRQn
+#define I2C_0_ERR_ISR       isr_i2c1_er
+/* I2C 0 pin configuration */
+#define I2C_0_PORT_CLKEN()  (RCC->AHBENR |= RCC_AHBENR_GPIOBEN)
+#define I2C_0_PORT          GPIOB
+#define I2C_0_SCL_PIN       8
+#define I2C_0_SCL_AF        4
+#define I2C_0_SDA_PIN       9
+#define I2C_0_SDA_AF        4
+
+/* I2C 1 device configuration */
+#define I2C_1_DEV           I2C2         /* Used for Sensors */
```

Comment:
Could you list the sensors? For me it's fine to do it without a description here. So: "*ST VL6180X, ST LSM6DS3, ST LIS3MDL, ST SLPS25H*"